Maquetado/scss/navbar.scss

```diff
@@ -2,6 +2,18 @@
   border-top: 7px solid $color;
 }
 
+@keyframes slidein {
+  from {
+    margin-left: 100%;
```

Comment:
Isn't a good practice anime this properties. Check this [article](https://medium.com/outsystems-experts/how-to-achieve-60-fps-animations-with-css3-db7b98610108)